ElasticBox/appliance-overview.md

```diff
@@ -7,15 +7,43 @@
 }}}
 
 ### ElasticBox Enterprise Appliance Overview
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.